### ISSUE TITLE
RF OOB error is not unbiased but at least not optimistically biased

### ISSUE THREAD

**Issue by manuelhelmerichs** [slds-lmu/lecture_i2ml#1212]
In Learning Goals v. slides-forests-intro: 

Explain this better: OOB error is not unbiased but (at least) not optimistically biased bc we do not violate the untouched test set principle. (von OL, heißt wenn resolved dort auch!)

**Comment by berndbischl** [comment]
i dont think we claim anything wrong anymore? @manuelhelmerichs ?

**Comment by manuelhelmerichs** [comment]
Yes, this has been corrected in the new forests-oob chapter ("As we do not violate the untouched test set principle, GE is not optimistically biased."), in the vid this is explained as well.
